Makefile

```diff
@@ -31,6 +31,7 @@ GETTING_STARTED_SCRIPT_DIR = website/content/en/preview/getting-started/getting-
 
 # Common Directories
 MOD_DIRS = $(shell find . -name go.mod -type f | xargs dirname)
+MOD_DIRS=`echo "${MOD_DIRS}" | sed 's/.\/website//g'`
```

Comment:
I don't think we want to remove it from the env var here, since we use this MOD_DIRS for the `make tidy` and `make download` target so that we automatically do the go mod tidy. Can you just do it in the `make verify` target?